lesson01/assignment/script-2.js

```diff
@@ -0,0 +1,108 @@
+// Search the "form", and handle its submit event
+const form = document.querySelector('#first-form');
+form.addEventListener('submit', function (event) {
+    event.preventDefault()
+    // create an array to transform or pass the user data, here an empty array
+    let data = [ 'love' , 'peace', '&', 'harmony']
+   
+    //Target the input field to get data from the user
+    const dataUser = document.querySelector('form .entry');
+
+    // Get the user's data
+    const grabDataUser = dataUser.value;
+
+    // Print the output in the form plus a text, and display the data string
+    document.querySelector('form span').innerHTML = `${'This is what you get '}` + ' " ' + grabDataUser + ' " ' + data.toString(); 

```

Comment:
This is a combination of string interpolation and concatenation syntaxes. It would be better to only use one:
```
`This is what you get ${grabDataUser} ${data.toString()}`
// or
'This is what you get ' +  grabDataUser  + data.toString()
```
More info: https://techaffinity.com/blog/javascript-string-interpolation/f